src/index.html

```diff
@@ -4,9 +4,434 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
```

Comment:
Generally you should add empty lines between siblings for readability:
```html
<body>
  <div>
    <span>1</span>
  
    <p>2</p>
  </div>

  <div>
    <div>
      <button></button>  
    </div>

    <span></span>
  </div>
</body>
```

But there are exceptions for example when you have a bunch of repeated tags in container like here:
```html
<ul>
  <li><a></a></li>
  <li><a></a></li>
  <li><a></a></li>
  <li><a></a></li>
</ul>

<ul>
  <li></li>
  <li></li>
  <li></li>
  <li></li>
</ul>
```
```suggestion
      <a href="#home" class="logo">
        <img
          src="images/logo.png"
          alt="logo"
          class="logo__img"
        >
      </a>
      
      <nav class="nav">
```